2-mandatory/4-space-colonies.js

```diff
@@ -15,7 +15,30 @@
   
 */
 
-function getSettlers() {}
+function likeThisPlanet(voyager){
+if(voyager.startsWith("A") && voyager.endsWith("family")) {
+  return true;
+  }else{
+    return false;
+  }
+  
+};
+
+function getSettlers(voyagers) {
+    let stayingFamiliesArray = voyagers.filter(likeThisPlanet);
+       return stayingFamiliesArray;
+};
+
```

Comment:
The logic is all correct. 
worth making another attemp, only one function getSettlers() and .include() after filter()